rules/eigen.json

```diff
@@ -13,6 +13,81 @@
           "distribution": "debian"
         }
       ]
+    },
+    {
+      "packages": ["eigen3-devel"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "fedora"
+        },
+        {
+          "os": "linux",
+          "distribution": "opensuse"
+        },
+        {
+          "os": "linux",
+          "distribution": "sle"
+        }
+      ]
+    },
+    {
+      "packages": ["eigen3-devel"],
+      "pre_install": [
+        {
+          "command": "rpm -q epel-release || yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
+        }
+      ],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "centos",
+          "versions": ["7"]
+        },
+        {
+          "os": "linux",
+          "distribution": "redhat",
+          "versions": ["7"]
+        }
+      ]
+    },
+    {
+      "packages": ["eigen3-devel"],
+      "pre_install": [
+        { "command": "dnf install -y dnf-plugins-core" },
+        { "command": "dnf config-manager --set-enabled powertools" }
```

Comment:
Yep, RHEL 8/9 follow Rocky Linux 8/9. We still call it `centos8` for backward compatibility.

There are just a few differences with what extra repos are available. Rocky 8 uses `powertools`, but RHEL 8 doesn't have this and uses CodeReady Builder instead. Here's an example of that you can copy: https://github.com/rstudio/r-system-requirements/blob/f2416f099eefeb4fa861677cab9163d3f7fd1fdc/rules/libgit2.json#L114-L128

For RHEL 9, the RHEL equivalent of Rocky's `crb` is also CodeReady Builder: https://github.com/rstudio/r-system-requirements/blob/f2416f099eefeb4fa861677cab9163d3f7fd1fdc/rules/libmagic.json#L78-L89